feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.html

```diff
@@ -26,27 +26,7 @@
       *ngFor="let item of attribute.values"
       [label]="item.valueDisplay"
       [selected]="item.selected"
-      [attr.aria-label]="
-        item.valuePrice && item.valuePrice?.value !== 0
-          ? item.valuePriceTotal && item.valuePriceTotal?.value !== 0
-            ? ('configurator.a11y.selectedValueOfAttributeFullWithPrice'
-              | cxTranslate
-                : {
-                    value: item.valueDisplay,
-                    attribute: attribute.label,
-                    price: item.valuePriceTotal.formattedValue
-                  })
-            : ('configurator.a11y.selectedValueOfAttributeFullWithPrice'
-              | cxTranslate
-                : {
-                    value: item.valueDisplay,
-                    attribute: attribute.label,
-                    price: item.valuePrice.formattedValue
-                  })
-          : ('configurator.a11y.selectedValueOfAttributeFull'
-            | cxTranslate
-              : { value: item.valueDisplay, attribute: attribute.label })
-      "
+      [attr.aria-label]="getAriaLabel(item, attribute)"
```

Comment:
@Changsuwan @ChristophHi:  should't this already be in develop? - can you please merged develop into this feature branch to get a clean dif. thx.